### DISCUSSION TITLE
Make camera sensor bounding box remain stationary, relative to users view.

### DISCUSSION THREAD

**Discussion by herboren** [Stellarium/stellarium#3193] [Ideas]
Stop forcing the bounding box to rotate when when panning the sphere. Allow bounding box of sensor to remain stationary, relative to the users view. 

![obs64_8TUsDDA49N](https://user-images.githubusercontent.com/987794/233765168-bd998864-9f74-4d16-82e9-b616a46deff6.gif)

Or provide a rotation option at the right angle of the bounding box

![image](https://user-images.githubusercontent.com/987794/233765217-27534207-95dd-4907-ad41-9e9ed0f86df5.png)

**Comment by 10110111** [comment]
Is it supposed to be an issue report? If yes, it should go to [Issues](https://github.com/Stellarium/stellarium/issues) section, with all the information requested by _New Issue_ template.

**Reply by herboren** [reply]
It is not an issue unless devs didn't intend it to be this way.

**Comment by alex-w** [comment]
> Stop forcing the bounding box to rotate when when panning the sphere. Allow bounding box of sensor to remain stationary, relative to the users view.

Why you using equatorial mount type for CCD-setup, when expected altazimuthal mount type?

**Reply by gzotti** [reply]
Yes, you are just using the wrong mount type in the Telescope config.
